elastic/kibana
# [Synthetics] Monitor details - add monitor pending UX

## Summary

Resolves https://github.com/elastic/kibana/issues/149418

[Monitor-Management-Synthetics---Kibana (7).webm](https://user-images.githubusercontent.com/11356435/228025027-199349d6-72e1-41d2-9656-2a6c5cefff31.webm)

Adds a pending monitor status for monitors that have not yet ran.

### Testing
1. Create a monitor and wait for some data to come back
2. Create a second monitor and ensure that it never runs. You can do this one of two ways: 1. Create a private location without an agent attached to it and assign the monitor to that location or 2. disable your connection to the service before creating the monitor, by providing the wrong credentials for the service in your kibana.dev.yml
3. Navigate to the first monitor. Ensure that the monitor details shows up appropriately after loading
4. Navigate to the second monitor. Ensure the pending monitors UX is displayed
